homeassistant/components/ness_alarm/alarm_control_panel.py

```diff
@@ -100,3 +105,15 @@ def _handle_arming_state_change(self, arming_state: ArmingState) -> None:
             _LOGGER.warning("Unhandled arming state: %s", arming_state)
 
         self.async_write_ha_state()
+
+
+def _get_state_for_arming_mode(arming_mode: ArmingMode):
+    arming_mode_to_state_map = {
+        ArmingMode.ARMED_AWAY: STATE_ALARM_ARMED_AWAY,
+        ArmingMode.ARMED_HOME: STATE_ALARM_ARMED_HOME,
+        ArmingMode.ARMED_DAY: STATE_ALARM_ARMED_AWAY,  # no applicable state, fallback to away
+        ArmingMode.ARMED_NIGHT: STATE_ALARM_ARMED_NIGHT,
+        ArmingMode.ARMED_VACATION: STATE_ALARM_ARMED_VACATION,
+        ArmingMode.ARMED_HIGHEST: STATE_ALARM_ARMED_AWAY,  # no applicable state, fallback to away
+    }
+    return arming_mode_to_state_map.get(arming_mode, STATE_ALARM_ARMED_AWAY)
```

Comment:
Make this a constant instead:
```suggestion
ARMING_MODE_TO_STATE_MAP = {
    ArmingMode.ARMED_AWAY: STATE_ALARM_ARMED_AWAY,
    ArmingMode.ARMED_HOME: STATE_ALARM_ARMED_HOME,
    ArmingMode.ARMED_DAY: STATE_ALARM_ARMED_AWAY,  # no applicable state, fallback to away
    ArmingMode.ARMED_NIGHT: STATE_ALARM_ARMED_NIGHT,
    ArmingMode.ARMED_VACATION: STATE_ALARM_ARMED_VACATION,
    ArmingMode.ARMED_HIGHEST: STATE_ALARM_ARMED_AWAY,  # no applicable state, fallback to away
}
```

Reply:
I'd prefer this, way cleaner and easy to read. I originally had it like this in my first commits, but found the tests failed in doing so since the static dependency on `ArmingMode` was made prior to `patch` setup in the tests, meaning the tests used `MockArmingMode`, but the production module depended on `ArmingMode` despite being patched.

I see your suggestion remove the mock enums, which should resolve this. I'll make those changes.